### ISSUE TITLE
Document Formatting

### ISSUE THREAD

**Issue by Tarkisflux** [SqueeG/awesomeTome#22]
I have some questions about some of the things that we're doing. Note, this may be technical in places (which I will mark with \begin{technicalbullshit} and \end{technicalbullshit} so that only @ExplosiveRunes and @Lokathor worry about it).

Anyway....

The formatting of the core books is to indent new paragraphs, including class features, except for the paragraph immediately following a header. Is there a reason we aren't using this formatting? It seems like it would help with ability start identification since we're not using alternative colors (due to weirdness with floating tables). I actually prefer to indent things in that style, so I'm wondering why it was changed.

Do we have a section numbering preference? We want numbering to display on chapters, but do we want number to display on class titles? On class subsections? Do we want some sort of parity between sections shown in the ToC and numbered sections? I don't know that the numbers are distracting, but I was thinking about it and wanted to bring it up.

\begin{technicalbullshit} 

Why are we using the memoir type instead of the book type? Don't know that I have a preference here, I'm really curious.

Can we not use the special color block formatting package @ExplosiveRunes? You could get similar results by declaring a tabularx or similar table with a single row and column using \rowcolor instead. It's not a big deal, but the current formatting complication seems unnecessary (and it also compiles badly on my setup for technical distro reasons that I can go into upon request).

 \end{technicalbullshit}

**Comment by ExplosiveRunes** [comment]
- **Sectioning:** I'd probably say it's only necessary to display chapter numbering in the PDF.
- **Memoir:** Honestly? I have no idea. I didn't even make the tome-ref-doc.tex file. I'd assume it's because Lokathor's SRD was memoir, and this would retain some of the basic formatting.
- **Blocks of Color:** We can get rid of it if you want. A tabularx environment provides similar functionality. Although, I must say, I've never quite liked how the text appeared within rows. If you look closely, it's not vertically centered. It's not _usually_ an issue, but I've run into instances where I've had text _touching_ the tops of the table. Commands that supposedly center text vertically in cells don't do shit for it. Maybe it's the font?

**Comment by Lokathor** [comment]
Note before I begin: I haven't been following the project too closely for the past week or so because it seemed like most discussion was having to do with what content would be in the work (skills and classes mostly). Not because I've lost interest in the project, but because I don't have much to say on the subject of what game material we're using or not.

> Paragraph Indentation Style

Yeah, we could do that style. No problem. The reason that we're _not currently_ doing that style has to do with the fact that in the Tomes themselves, and in After Sundown and Ends of The Matrix, all those little taglines at the start of sections would count as the first paragraph and "eat up" the non-indentation anyway. This could have been worked around, but long ago I just decided to have all paragraphs be indented all the time because I like the indentation. We could go back to "first paragraph isn't indented" though if folks want.

> Section Numbering

It's simple enough to just number all chapters and sections, but not subsections and other divisions "below" the section level. It'd be possible to leave off section numbering from class declarations or skill declarations or whatnot while having it on other sections, if that was really desired. I think it looks fine to have section numbers next to class titles though, personally.

> Memoir

It's kinda like "book" but with some more advanced features and such built in. I think I first began using it instead of book because it lets you easily declare the stock size. In general it's specifically designed towards ease of use and "good defaults" in places. It has a very huge and detailed reference manual, if you'd like to look into it more. We could go to Book if we needed to I guess. There's a large memoir section on Stack Exchange and stuff, it's popular enough that we're not shooting ourselves in the foot by using Memoir.

> Color

I don't care what package we're doing it with, but I think that our colors should be actual colors and not just greyscale. It could be all along a single color scale, or more than one color, but not all grey.

**Comment by Tarkisflux** [comment]
Paragraphs:

I don't care if they're all indented and the tagline eats the non-indent (though I would prefer to do tagline and then non-indent if we can swing it), but the class features all have \noindent on them that kills the indent entirely. I was mostly asking why we're doing that, as I'd like to get rid of them and go with regular indents.

**Comment by ExplosiveRunes** [comment]
@Lokathor On the colors, I just chose the grays as examples. The way I set up the formatting I didn't use any explicit colors, but instead defined new colors (colorone, colortwo, headercolor, tablecolorone, tablecolortwo) that get used. By altering the definition of those colors, the changes will cascade to all the tables. I was working under the assumption that we'd eventually have some sort of basic background with page borders to make it look more like a traditional RPG book, and that we'd choose the colors to work with that.

Edit: Oh, the noindent on the class features. That was a holdover from awesomeTome, and is built into the \ability command which was lifted from it directly. I honestly think it looks a little better like that, but I honestly don't care that much either way.

**Comment by Tarkisflux** [comment]
Noindent works great when you have empty lines trailing paragraphs or features to indicate breaks. Since we don't have those lines (and I don't want them), I'm with Lokathor on liking indents.

**Comment by ExplosiveRunes** [comment]
I liked how \tabulinesep=1mm ended up looking when you went back over the weapon tables. I decided to do an experiment and do _all_ the tables in the equipment chapter that way (we can always switch it back). It makes the tables take up more space, but they're certainly nicer to look at. Is this something we want to do throughout the document? There's certainly not a tabular anywhere that couldn't be quickly converted to a tabu.

**Comment by Tarkisflux** [comment]
Yes. I was planning on getting to it when I stripped the formatting out of the main document and put it in chapter specific files instead, but if you want to convert already to see how it looks feel free.

Why are some tables (and footnotes, and table titles) rendered in a different font than others in the equipment section?
